### PR TITLE
Home page of ecommerce website

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my home page

live link: https://sportssite.netlify.app/index.html

f7
f8

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (khushijohri2001) [inline index.html]: you can add meta tags for author and description. but its not mandatory

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (khushijohri2001) [inline index.html]: use a proper title for the page

File: index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                <i class="fa-solid fa-star"></i>
+
+                <i class="fa-solid fa-star-o"></i>
+            </div>
+            <p>Rs. 10,539</p>
+        </div>
+
+        
+        </div>
+
```
Comment (khushijohri2001) [inline index.html]: too much extra space can be remove. will make your code look clean

**Review by khushijohri2001 — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+
+
+       
+        </div>
+
+
+
+
```
Comment (i-am-a-codegirl) [inline index.html]: remove extra spaces

File: style.css [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+}
+.categories{
+    margin: 70px 0;
+
+}
+/* .col-3{
+    flex-basis: 30%;
+    /* min-width: 250px;
+    margin-bottom: 30px; */
+/* .small-container{
```
Comment (i-am-a-codegirl) [inline style.css]: remove comments that are useless

**Review by i-am-a-codegirl — APPROVED** [review]
Looks good to merge just view my suggestions

**Pull request merged**